### ISSUE TITLE
Rectify 11 3.0 Completely Removes All Access To The Windows Control Panel

### ISSUE THREAD

**Issue by soulcollectivex** [Rectify11/Installer#319]
### Description

I updated from Rectify 11 v3.0 RC3 to v3.0 (newest release) and it removed the entire control panel from my Windows 11 install, and all of my restore points were deleted somehow also. Absolutely no idea why this happened... I uploaded a screenshot below of what happens when trying to access the control panel after installing this new update as well as my installer log... thanks for any help, I appreciate it.
<img width="373" alt="Screenshot 2023-10-15 132243" src="https://github.com/Rectify11/Installer/assets/126836951/1431bcc8-8d10-4a72-891c-d7ae5ce3a289">

### Steps To Reproduce

Have Rectify11 v.30 RC3 installed with System Restore Backups, and then install the new v3.0 update. 

### Rectify11 Version

Version 3.0

### Windows Version

Windows 11 Pro Version 22H2 (0S Build 22621.2428)

### Logs

[installer.log](https://github.com/Rectify11/Installer/files/12910270/installer.log)

**Comment by SGF1760** [comment]
Looking at your log file there is one warning that could explain it. but it might be for something else. Using it i'd have to guess while installing themes it had an error modifying control panel.
Because you have no restore points, your better off backing up everything important and reinstalling fresh windows 11.

**Comment by soulcollectivex** [comment]
Thanks for replying, I appreciate your help.
I have rebooted my system and the issue persisted.
Another odd issue with my problem is that I actually did have restore points that created myself, and after updating rectify 11 they were deleted...
I believe I may have resolved the issue though. I have another program called windhawk which loads mods into Windows explorer, and I think this is most likely what has caused the issue, because after exiting it the control panel opens just fine.
